docs/03-dev-howtos/18-working-with-emails.md

```diff
@@ -89,3 +89,40 @@ function validate(form) {
 - https://davidwalsh.name/html5-email
 
 4) If all else fails, ExactTarget has checks in place for valid emails and also cleans the lists.
+
+## Email rendering
+
+Fronts and articles can be rendered in email-friendly HTML by appending the URL parameter `format=email`. The response from these endpoints can be put into an email body and will render well in a wide variety of email clients.
```

Comment:
Good to have this information! Who is the target audience though? The email sign up page stuff seems to be pretty detailed info for devs so I'm wondering if this is the place for this info.